modules/multiauth/docs/multiauth.md

```diff
@@ -77,7 +79,7 @@ compatible fashion so both cases should work.
 
 Each source in the sources array has a key and a value. As
 mentioned above the key is the authsource identifier and the value
-is another array with two optional keys: 'text' and 'css-class'.
+is another array with optional keys: 'text', 'css-class', 'help', and 'ContextClassRef'.
```

Comment:
You're missing `Authn`

Reply:
Initially I implemented it with AuthnContextClassRef. But it seemed weird to reference Authn in the multiauth config. So I changed it to ContextClassRef. I've reverted back to AuthnContextClassRef.